### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by RobbyThomas2525** [iBicha/playlet#480] [Ideas]
just wondering if there's any plans for playlet to work on a desktop web browser without the qr code and is there any plans for a ios or android app

**Comment by iBicha** [comment]
> just wondering if there's any plans for playlet to work on a desktop web browser without the qr code

Under the qr code there should be a url written on screen (http://192.168.x.x:8888 or similar) if you open that link in your desktop web browser (given you are connected to the same network/wifi as the tv) then the web app should work fine on desktop as well

> is there any plans for a ios or android app

Playlet will always be primarily a Roku app, so no plan for a standalone mobile app, since there are alternative (e.g. [NewPipe](https://github.com/TeamNewPipe/NewPipe)). 
The web app was created to facilitate casting from phone, and there was considerations to create a mobile app just to help with browsing/casting. But since Playlet supports casting from official YouTube app, then there was no justification to create an app just to cast.

I hope this answers your question
